blog/2024-05-02-react-sdk-release.md

```diff
@@ -0,0 +1,118 @@
+---
+slug: "openfeature-react-sdk-release"
+title: "OpenFeature React SDK Release"
+date: 2024-05-02
+authors: [toddbaert]
+description: "Announcing the first official release of the OpenFeature React SDK"
+tags: [react, client, sdk, web, javascript, typescript]
+draft: false
+---
+
+We're stoked to announce the first release of the [@openfeature/react-sdk](https://www.npmjs.com/package/@openfeature/react-sdk)!
```

Comment:
I think we should link here instead of NPM.

https://deploy-preview-541--openfeature.netlify.app/docs/reference/technologies/client/web/react

Reply:
Agreed